### PR TITLE
Update from ohm to bricks

### PR DESCRIPTION
**Pull request by 0xtaichi**
## Updates
1. Update from ohm to bricks
2. Add circulatingSupply functions
3. Remove all unused methods and constants

### CODE REVIEW

**Comment by 0xkowloon** [comment]
please no `.DS_Store` in the pull request, put it in `.gitignore`

File: abis/FRAX.json [hunk 1]
```diff
@@ -0,0 +1,595 @@
+[
```
Comment (0xkowloon) [inline abis/FRAX.json]: Would a standard ERC20 ABI work? Do we need a Frax specific ABI?

File: abis/ReserveBrickFraxContract.json [hunk 2]
```diff
@@ -0,0 +1,345 @@
+[
```
Comment (0xkowloon) [inline abis/ReserveBrickFraxContract.json]: is this the same or different from the `FRAX.json` ABI

File: abis/WrappedToken.json [hunk 3]
```diff
@@ -0,0 +1,385 @@
+[
```
Comment (0xkowloon) [inline abis/WrappedToken.json:1]: Just to confirm, did you pull this ABI from gm-contracts or from the actual WETH or WFTM contract on ETH/FTM mainnet? If this is from gm-contracts it is more accurate to pull it from mainnet instead

File: generated/BrickFraxBond/CirculatingSupply.ts [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+    if (result.reverted) {
+      return new ethereum.CallResult();
+    }
+    let value = result.value;
+    return ethereum.CallResult.fromValue(value[0].toAddress());
+  }
+
+  OHMCirculatingSupply(): BigInt {
+    let result = super.call(
+      "OHMCirculatingSupply",
```
Comment (0xkowloon) [inline generated/BrickFraxBond/CirculatingSupply.ts]: We are not using this contract right?

File: generated/BrickFraxBond/ConvexAllocator.ts [hunk 5]
```diff
@@ -0,0 +1,687 @@
+// THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+
+import {
```
Comment (0xkowloon) [inline generated/BrickFraxBond/ConvexAllocator.ts]: we are not using this contract right?

File: schema.graphql [hunk 6]
```diff
@@ -154,10 +154,10 @@ type BondDiscount @entity {
   id: ID!
   timestamp: BigInt!
   dai_discount: BigDecimal!
-  ohmdai_discount: BigDecimal!
+  brickdai_discount: BigDecimal!
```
Comment (0xkowloon) [inline schema.graphql]: there is no brickdai and bricklusd right?

File: src/utils/BRICKie.ts [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+        holders.value = holders.value.plus(BigInt.fromI32(1))
+        holders.save()
+        brickie.active = true
+    }
+
+    let bonds = balance.bonds
+
+    //BRICK-FRAX
+    if(transaction.blockNumber.gt(BigInt.fromString(BRICKFRAXLPBOND_CONTRACT1_BLOCK))){
+        let bondFRAXDai_contract = BrickFraxBond.bind(Address.fromString(BRICKFRAXLPBOND_CONTRACT1))
```
Comment (0xkowloon) [inline src/utils/BRICKie.ts]: bondFRAXDai_contract? DAI?

File: src/utils/BRICKie.ts [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+        let pending = bondFRAXDai_contract.bondInfo(Address.fromString(brickie.id))
+        if (pending.value1.gt(BigInt.fromString("0"))){
+            let pending_bond = toDecimal(pending.value1, 9)
+            balance.bondBalance = balance.bondBalance.plus(pending_bond)
+
+            let binfo = loadOrCreateContractInfo(brickie.id + transaction.timestamp.toString() + "BrickFraxBondV1")
+            binfo.name = "DAI"
```
Comment (0xkowloon) [inline src/utils/BRICKie.ts]: DAI?

File: src/utils/BRICKie.ts [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+            binfo.amount = pending_bond
+            binfo.save()
+            bonds!.push(binfo.id)
+
+            log.debug("Brickie {} pending ETHBondV1 V1 {} on tx {}", [brickie.id, toDecimal(pending.value1, 9).toString(), transaction.id])
+        }
+    }
+    balance.bonds = bonds
+
+    //TODO add LUSD and BRICKLUSD
```
Comment (0xkowloon) [inline src/utils/BRICKie.ts]: ?

File: src/utils/Transactions.ts [hunk 10]
```diff
@@ -12,7 +12,7 @@ export function loadOrCreateTransaction(eth_transaction : ethereum.Transaction,
     transaction.from = eth_transaction.from
     transaction.to = eth_transaction.to
     transaction.value = eth_transaction.value
-    transaction.gasUsed = eth_transaction.gasUsed
+    // transaction.gasUsed = eth_transaction.gasUsed
```
Comment (0xkowloon) [inline src/utils/Transactions.ts]: why is this commented out

File: abis/FRAX.json (hunk 1 above)
Comment (0xtaichi) [reply abis/FRAX.json]: yes, standard ERC20 should be good enough

File: abis/ReserveBrickFraxContract.json (hunk 2 above)
Comment (0xtaichi) [reply abis/ReserveBrickFraxContract.json]: it is different, can remove tho since not being used

File: abis/WrappedToken.json (hunk 3 above)
Comment (0xtaichi) [reply abis/WrappedToken.json:1]: pull it from mainnet then

Comment (0xtaichi) [reply abis/WrappedToken.json:1]: will pull it from mainnet then

File: abis/ReserveBrickFraxContract.json (hunk 2 above)
Comment (0xtaichi) [reply abis/ReserveBrickFraxContract.json:1]: it is different, can remove tho since not being used

File: abis/WrappedToken.json (hunk 3 above)
Comment (0xtaichi) [reply abis/WrappedToken.json:1]: will pull it from mainnet then

File: generated/BrickFraxBond/CirculatingSupply.ts (hunk 4 above)
Comment (0xtaichi) [reply generated/BrickFraxBond/CirculatingSupply.ts]: yes, can be removed instead

Comment (0xtaichi) [reply generated/BrickFraxBond/CirculatingSupply.ts]: yes, can be removed instead

File: generated/BrickFraxBond/ConvexAllocator.ts (hunk 5 above)
Comment (0xtaichi) [reply generated/BrickFraxBond/ConvexAllocator.ts]: yes, can be removed instead

File: schema.graphql (hunk 6 above)
Comment (0xtaichi) [reply schema.graphql]: yes, can be removed instead

File: src/utils/BRICKie.ts (hunk 7 above)
Comment (0xtaichi) [reply src/utils/BRICKie.ts]: oh will rename it

File: src/utils/BRICKie.ts (hunk 8 above)
Comment (0xtaichi) [reply src/utils/BRICKie.ts]: oh will rename it

File: src/utils/BRICKie.ts (hunk 9 above)
Comment (0xtaichi) [reply src/utils/BRICKie.ts]: will remove the TODO

File: src/utils/Transactions.ts (hunk 10 above)
Comment (0xtaichi) [reply src/utils/Transactions.ts]: according to the latest version of `@graphprotocol/graph-ts`, should use `gasLimit` instead
https://github.com/graphprotocol/graph-node/issues/2619

**Comment by 0xkowloon** [comment]
@0xtaichi the last commit LGTM

**Pull request merged**